src/app/books/interfaces/book.interface.ts

```diff
@@ -1,8 +1,10 @@
-import { IGenre } from '../../genres';
+import { IGenre } from '@genres/.';
+import { IAuthor } from '@authors/.';
 
 export interface IBook {
   id: number;
   description: string;
+  author: IAuthor;
```

Comment:
Are you sure that you need `author` and `author_id` properties at the same time?